en/docs/tutorials/integration-tutorials/service-catalog-tutorial-for-proxy-services.md

```diff
@@ -232,7 +232,9 @@ Package the artifacts in your composite exporter module to be able to deploy the
 
     -   `StockQuoteProxy`
 
-3.  Save the changes.
+3. By Default, `Publish to Service Catalog` check box is enabled. If not, please enable the check box in the wizard so that it will include Metadata files of the selected artifacts.
```

Comment:
```suggestion
3. By default, the `Publish to Service Catalog` checkbox is enabled. If not, please select the checkbox in the wizard so that it will include metadata files of the selected artifacts.
```